### DISCUSSION TITLE
Contributing features

### DISCUSSION THREAD

**Discussion by Samantha-uk** [alf45tar/PedalinoMini#343] [General]
Continue to be impressed with this project; hence "sponsoring" to get access to the sponsors version.

As with almost all software, there are somethings I would like that are not there.
Oh the joy of OSS ... want it? ... add it! :)

I have tweaked the codebase a little to add new board support and also changed some pin assignments.  I am considering a deeper dive to add support for multiple OLED displays; the h/w change required is not very difficult and, **for me**, it would be really nice to have scribble strip displays on a per switch or per switch pair basis.

My starting point would be the sponsors codebase.  I cannot fork it; not a huge problem.

But how would I submit a PR based on any changes I made?

**Comment by alf45tar** [comment]
Every contributions are welcome. We will find the way.
